src/test/scala/strawman/collection/test/TraverseTest.scala

```diff
@@ -29,14 +30,21 @@ class TraverseTest {
   def traverseTest: Unit = {
 
     val xs1 = immutable.List(Some(1), None, Some(2))
-    optionSequence(xs1)(immutable.List.canBuild[Int]) // TODO Remove explicit CanBuild parameter after https://issues.scala-lang.org/browse/SI-10081 is fixed?
+    val o1 = optionSequence(xs1)
+    val o1t: Option[immutable.List[Int]] = o1
 
     val xs2 = immutable.TreeSet(Some("foo"), Some("bar"), None)
-    optionSequence(xs2)(immutable.TreeSet.canBuild[String])
+    val o2 = optionSequence(xs2)
+    val o2t: Option[immutable.TreeSet[String]] = o2
 
     val xs3 = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
-    eitherSequence(xs3)(mutable.ListBuffer.canBuild[String])
+    val e1 = eitherSequence(xs3)
+    val e1t: Either[Int, mutable.ListBuffer[String]] = e1
 
+    // This use case from https://github.com/scala/scala/pull/5233 still eludes us
```

Comment:
I'm considering to resurrect the idea of path-dependent types for `CanBuild` scoping, essentially unifying the new way I implement `to` (which doesn't need to infer a unary type constructor anymore) with the implicit builder factories. This would not enable a type-driven `breakOut` but a value-driven approach with an explicit factory similar to the syntax below (like in `to`).